dll/win32/shdocvw/shdocvw.h

```diff
@@ -46,13 +46,22 @@ extern HRESULT SHDOCVW_GetShellInstanceObjectClassObject(REFCLSID rclsid,
 /**********************************************************************
  * Dll lifetime tracking declaration for shdocvw.dll
  */
+#ifdef __REACTOS__
+# ifdef __cplusplus
```

Comment:
Would using EXTERN_C unconditionally (even when it's C code) fail to compile?

Reply:
It fails to compile on simple form. This is a possible workaround.